File: C++/time.cpp
#include <iostream>

int main(){
    int n = 0;
    std::cin >> n;
    for (int i = 0; i < n; i++){
        int num = 0;
        std::cin >> num;
        std::cout << ((num % 100 < 60 && num / 100 < 24) ? "YES " : "NO ");
    }
}

File: Golang/time.go
package main

import(
	"fmt"
)

func main(){
	var n int = 0
	fmt.Scanf("%d", &n)
	for i := 0; i < n; i++{
		var num int = 0
		fmt.Scanf("%d", &num)
		if num % 100 < 60 && num / 100 < 24{
			fmt.Printf("YES ")
		} else{
			fmt.Printf("NO ")
		}
	}
}